iam-login-service/src/main/resources/application-saml.yml

```diff
@@ -4,15 +4,23 @@ saml:
   keystore-password: ${IAM_SAML_KEYSTORE_PASSWORD:password}
   key-id: ${IAM_SAML_KEY_ID:iam}
   key-password: ${IAM_SAML_KEY_PASSWORD:password}
-  idp-metadata: ${IAM_SAML_IDP_METADATA:classpath:/saml/idp-metadata.xml}
   max-assertion-time-sec: ${IAM_SAML_MAX_ASSERTION_TIME:3000}
   max-authentication-age-sec: ${IAM_SAML_MAX_AUTHENTICATION_AGE:86400}
   id-resolvers: ${IAM_SAML_ID_RESOLVERS:eduPersonUniqueId,eduPersonTargetedId,eduPersonPrincipalName}
   metadata-lookup-service-refresh-period-sec: ${IAM_SAML_METADATA_LOOKUP_SERVICE_REFRESH_PERIOD_SEC:600}
+  login-button-text: ${IAM_SAML_LOGIN_BUTTON_TEXT:"Sign in with SAML"}
```

Comment:
Other buttons don't have double quotes into the button text: it's not a problem, but the style is different.